css-brochure-form/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Brochure Form</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
+</head>
+<body>
+<h1>Learn More</h1>
+<form>
+  <label for="user_email">Company Email</label>
+  <div></div>
```

Comment:
What's the point of this empty `div`? It has no content inside, therefore it has no purpose. If you want the `label` to be on it's own line, then you should be giving the `label` a `display` value that makes it take up the full width. Think block vs inline.

If you're using it for spacing purposes, then you should be using `margin` instead